src/components/FontUploader.tsx

```diff
@@ -1,14 +1,16 @@
 import React, { useCallback, useState } from 'react';
 import '../styles/FontUploader.css';
 import { useDropzone } from 'react-dropzone';
+import { Typr } from 'typr-ts';
 
 const FontUploader: React.FC<{ onFontSelected: (selectedFont: File | null) => void }> = ({
   onFontSelected,
 }) => {
   const [, setSelectedFont] = useState<File | null>(null);
   const [, setFontPreview] = useState<string | null>(null);
-  const initialTxt = 'Drag and drop font files here';
-  const [text, setText] = React.useState(initialTxt);
+  const initialFontName = 'Drag and drop font files here';
```

Comment:
We only accept one file per drop zone, right?

```suggestion
  const initialFontName = 'Drop a font here';
```